Display/Documentation/HowToTurnOn.md

```diff
@@ -0,0 +1,28 @@
+How to turn on the Tetra Ventillator Splitter
+=============================================
+
+1. Ensure that the splitter is connected to a power supply
+
+2. Ensure that the airlines connecting the splitter and ventillator are attached (detailed in a serperate document)
+
+3. Flip the on-off switch to on
+
+4. After a short boot-up sequence, a screen showing data for each of the patient airlines should appear, which looks like this
+
+   ![Airline data information screen](https://cdn.discordapp.com/attachments/718703783730741290/720436682851352697/VentilatorDisplay_1.jpeg)
+
+   - If instead a computer desktop appears, connect a mouse to the USB port on the splitter and double-click on the "Tetra Splitter Data Readout" icon
+
+   - If there is no such icon, ensure that the Tetra Ventillator Splitter software is installed
+
+5. Verify that all the air connections are good
+
+   1. Plug the ends of all patient connections
+
+   2. Begin running the ventillator
+
+   3. The numbers on the Airline Data Information Screen should be changing
+
+      - If the numbers are not changing, make sure the stop valves for each airline is open
```

Comment:
So at first, two patients will be on, then a third will be added, then a fourth, then one patient will come off, then another added, then another two come off, etc-- so we should make sure that the system can function just fine with some lines working and others shut off.

Reply:
Sure thing